libs/wingc/src/lsp/snapshots/completions/empty.snap

```diff
@@ -0,0 +1,27 @@
+---
+source: libs/wingc/src/lsp/completions.rs
+---
+- label: assert
+  kind: 3
+  detail: "(bool): void"
+  insertText: assert($0)
+  insertTextFormat: 2
```

Comment:
What do these mean? Maybe we can add some details to the contributing guide about it?

I wonder if there should be an option to just take snapshot of the labels vs snapshots of the entire struct, depending on the unit test

Reply:
I can add a comment referencing the original type which can provide info.

I think all of these are somewhat important, although the enums look useless since they're just numbers. If there is a test where we should limit what gets snapshotted it shouldn't be too hard to add.